docs/record_versioning.md

```diff
@@ -0,0 +1,54 @@
+# Record Versioning
+
+The PaperTrail gem is a Ruby on Rails plugin that helps you keep a record of how your data is changing. It creates a version history for your models, tracking changes to them over time. This can be useful for auditing or versioning purposes.
```

Comment:
let's link the word paperTrail to the plugin page so they can quickly jump to it from the first sentence itself